esmvaltool/diag_scripts/shared/plot/zonalmean_profile.ncl

```diff
@@ -105,6 +105,22 @@ begin
   plot = gsn_csm_pres_hgt(wks, data, res)
   plot@outfile = wks@name
 
+  ; Add the outline of the climatological tropopause
+  if (isatt(data, "add_tropopause")) then
```

Comment:
Could be removed if you set a default value of the parameter in main.ncl (see my recommendation below).

Reply:
Can zonalmean_profile.ncl be used by another diagnostic which is not providing such a default value?

Reply:
> Can zonalmean_profile.ncl be used by another diagnostic which is not providing such a default value?

Yes, it can! You are absolutely right, the check if the attribute "add_tropopause" is set needs to stay in!